### PR TITLE
Fixed firstpage:removed the submit button

### LINKED ISSUE #12: remove submit from fast page

### PR DESCRIPTION
**Pull request by harshad16**
Fixes #12
Removed the submit from first page.
Fixed the issue in first page.

### CODE REVIEW

File: FirstPage.html [hunk 1]
```diff
@@ -12,6 +12,13 @@
 </div>
 
 
+  <!-- scripts -->
+  <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
```
Comment (dmk1111) [inline FirstPage.html]: Why do you need to bring jQuery into the project if you don't use it?

File: FirstPage.html [hunk 2]
```diff
@@ -12,6 +12,13 @@
...
+  <script type="text/javascript">
+    function myFunction() {
```
Comment (dmk1111) [inline FirstPage.html]: Consider renaming `myFunction` to something more meaningful

File: FirstPage.html (hunk 1 above)
Comment (rajibmitra) [reply FirstPage.html]: agree.

Comment (harshad16) [reply FirstPage.html]: Removed it 👍

File: FirstPage.html (hunk 2 above)
Comment (harshad16) [reply FirstPage.html]: Made the necessary changes! 👍

**Comment by harshad16** [comment]
Made the necessary changes! 👍

**Pull request merged**